velox/core/QueryConfig.h

```diff
@@ -94,11 +94,16 @@ class QueryConfig {
   static constexpr const char* kMaxPartitionedOutputBufferSize =
       "driver.max-page-partitioning-buffer-size";
 
-  /// Preffered number of rows to be returned by operators from
+  /// Preferred number of rows to be returned by operators from
   /// Operator::getOutput.
   static constexpr const char* kPreferredOutputBatchSize =
       "preferred_output_batch_size";
 
+  /// Preferred number of  bytes to be returned by operators from
+  /// Operator::getOutput.
+  static constexpr const char* kPreferredOutputBatchByteSize =
```

Comment:
Would you extract this change into a separate PR and add it to documentation?

Reply:
Will do per offline discussion.